test/utils/smoke-test-list.ts

```diff
@@ -14,20 +14,20 @@ export enum AutorestParams {
 
 const getArmReadmes = (): SpecDefinition[] => {
   const armTags = [
-    "package-features-2015-12",
-    "package-locks-2016-09",
-    "package-policy-2019-09",
+    // "package-features-2015-12",
+    // "package-locks-2016-09",
+    // "package-policy-2019-09",
     "package-resources-2019-08",
-    "package-subscriptions-2019-06",
```

Comment:
Any specific reason these were removed?

Reply:
this was a mistake thanks for catching it!